server/src/main/java/org/apache/cassandra/sidecar/handlers/NativeUpdateHandler.java

```diff
@@ -0,0 +1,99 @@
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.handlers;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.inject.Inject;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.net.SocketAddress;
+import io.vertx.ext.auth.authorization.Authorization;
+import io.vertx.ext.web.RoutingContext;
+import org.apache.cassandra.sidecar.acl.authorization.BasicPermissions;
+import org.apache.cassandra.sidecar.common.request.data.NodeCommandRequestPayload;
+import org.apache.cassandra.sidecar.common.server.StorageOperations;
+import org.apache.cassandra.sidecar.concurrent.ExecutorPools;
+import org.apache.cassandra.sidecar.utils.InstanceMetadataFetcher;
+import org.jetbrains.annotations.NotNull;
+
+
+/**
+ * Handler for starting or stopping native transport on a Cassandra node.
+ * <p>
+ * Expects a JSON body:
+ * { "state": "start" }  or  { "state": "stop" }
+ * </p>
+ */
+public class NativeUpdateHandler extends NodeCommandHandler implements AccessProtected
+{
+    private final ExecutorPools executorPools;
+    private static final Logger LOGGER = LoggerFactory.getLogger(NativeUpdateHandler.class);
+
+
+    @Inject
+    public NativeUpdateHandler(InstanceMetadataFetcher metadataFetcher,
+                               ExecutorPools executorPools)
+    {
+        super(metadataFetcher, executorPools, null);
+        this.executorPools = executorPools;
+    }
+
+    @Override
+    public Set<Authorization> requiredAuthorizations()
+    {
+        return Collections.singleton(BasicPermissions.WRITE_NATIVE.toAuthorization());
+    }
+
+    @Override
+    protected void handleInternal(RoutingContext context,
+                                  HttpServerRequest httpRequest,
+                                  @NotNull String host,
+                                  SocketAddress remoteAddress,
+                                  NodeCommandRequestPayload request)
+    {
+        StorageOperations storageOps = metadataFetcher.delegate(host).storageOperations();
+
+        executorPools.service()
+                     .runBlocking(() -> {
+                         switch (request.getState())
+                         {
+                             case START:
+                                 storageOps.startNativeTransport();
+                                 break;
+                             case STOP:
+                                 storageOps.stopNativeTransport();
+                                 break;
+                             default:
+                                 throw new IllegalStateException("Unknown state: " + request.getState());
+                         }
+                     })
+                     .onSuccess(ignored -> {
+                         context.response().setStatusCode(HttpResponseStatus.ACCEPTED.code()).end();
```

Comment:
let's respond with a json payload, status: OK is acceptable, or something along those lines